### PR TITLE
Test for nncf vs keras.applications model graph equality

### PR DESCRIPTION
**Pull request by negvet**
- Added test which compares nncf-defined models and keras.applications models
- Models which have been updated in NNCF (minor changes) to pass the test: InceptionResNetV2, ResNet50V2, MobileNet v1, and v2

### CODE REVIEW

File: examples/tensorflow/common/models.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def MobileNetV3(stack_fn, last_point_ch, input_shape=None, model_type='large', *
     se_ratio = 0.25
 
     x = img_input
-    x = layers.Rescaling(scale=1. / 127.5, offset=-1.)(x)
+    x = layers.Rescaling(1. / 255.)(x)
```
Comment (alexsu52) [inline examples/tensorflow/common/models.py:64]: These changes will break accuracy for mobilenet_v3. Please, revert these changes.

File: examples/tensorflow/common/models.py [hunk 2]
```diff
@@ -103,6 +96,12 @@ def MobileNetV3(stack_fn, last_point_ch, input_shape=None, model_type='large', *
         name='Conv_2')(x)
     x = activation(x)
 
+    x = layers.GlobalAveragePooling2D()(x)
```
Comment (alexsu52) [inline examples/tensorflow/common/models.py:99]: These changes will break accuracy for mobilenet_v3. Please, revert these changes.

**Review by alexsu52 — CHANGES_REQUESTED** [review]
Please remove from the PR test files ' .../tensorflow/runs/'

File: examples/tensorflow/common/models.py (hunk 1 above)
Comment (negvet) [reply examples/tensorflow/common/models.py]: Reverted.

File: examples/tensorflow/common/models.py (hunk 2 above)
Comment (negvet) [reply examples/tensorflow/common/models.py]: Reverted.

**Review by andrey-churkin — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]
